### PR TITLE
Jetpack Plans: Add links for “Enable Manage” card

### PR DESCRIPTION
**Pull request by ryelle**
During the install flow, if you don't have manage enabled, we show a card asking you to enable it. This adds the relevant links :)

To test:

- Visit the installer for a site without manage (/plugins/setup/$site)
- See that buttons "Manual Installation" "Enable Manage" will take you somewhere (support and wp-admin, respectively)

cc @roccotripaldi @johnHackworth 

Test live: https://calypso.live/?branch=add/jetpack-manage-links

### CODE REVIEW

**Comment by roccotripaldi** [comment]
Wow. You are on a roll. I've gotta duck out for a diner date, but I can review this one later tonight.

File: client/my-sites/plugins/jetpack-plugins-setup/index.jsx [hunk 1]
```diff
@@ -310,8 +311,8 @@ const PlansSetup = React.createClass( {
 							components: { strong: <strong /> }
 						} )
 					}</p>
-					<Button primary>Enable Manage</Button>
-					<Button>Manual Installation</Button>
+					<Button primary href={ manageUrl }>Enable Manage</Button>
```
Comment (mtias) [inline client/my-sites/plugins/jetpack-plugins-setup/index.jsx]: Should these be translated?

**Comment by johnHackworth** [comment]
Good point  @mtias 

I fixed that, and I also changed where we get our selectedSite. We were using the redux sites subtree, but I change it to use the good (ha!) old sites-list, because we need the sites to contain their jetpack modules list

**Comment by mtias** [comment]
@johnHackworth you could work on porting such a selector 😉

**Comment by johnHackworth** [comment]
indeed :D 

but that commit would end being like 6 times the size of this PR and I would be totally sidetracking @ryelle's original intent, so let's do it in another branch :)

**Comment by mtias** [comment]
oh, sure, separately sound fine :)

**Comment by ryelle** [comment]
I had a separate PR for the selectedSite to get the modules list (#5905), but I think this is a better approach.

@johnHackworth is this ready to merge?

**Comment by roccotripaldi** [comment]
This works as described. Both the "Manual Installation" and "Enable Manage" are linked as described. 
The "Enable Manage" link allows someone to enable manage, but the redirection is wonky. That will need to be fixed in Jetpack. So I think this PR is good to go!

**Pull request merged**